addons/mail/static/src/models/message/message.js

```diff
@@ -583,10 +583,19 @@ function factory(dependencies) {
         body: attr({
             default: "",
         }),
+        /**
+         * Store the channel ids needed to create a message with postMessage.
+         */
+        channel_ids: attr({
```

Comment:
Sorry my previous comment here was wrong actually, we still want to support sending channel_ids because it is for mentions, so this needs to be reintroduced, but with a better field name and doc.

Reply:
Wait no I'm just confusing myself now, `channel_ids` was indeed removed from message post. We can still mention channel but this is only done with a special link in the message, no need to store or send the value.